styles/starwarsffg.css

```diff
@@ -2379,3 +2379,52 @@ img {
   min-width: 48%;
   margin: 5px 2px;
 }
+
+/* Destiny Pool UI Display */
```

Comment:
These will get overwritten when the sass files are compiled.  They will need to be moved into an .scss file, probably the scss/componentas/groupmanager.scss or create a new one and reference it in the scss/starwarsffg.scss